onprc_ehr/resources/queries/study/treatmentSchedule.sql

```diff
@@ -84,8 +84,9 @@ SELECT
 FROM ehr_lookups.dateRange dr
 
 JOIN study."Treatment Orders" t1
-  --NOTE: should the enddate consider date/time?
-  ON (dr.dateOnly >= t1.dateOnly and dr.dateOnly <= t1.enddateCoalesced AND
+  --COALESCE returns results for treatment orders without an enddate and with s.date in the future. This is needed for
+  --printable report Clinical Medications
+  ON (dr.dateOnly >= t1.dateOnly AND dr.dateOnly <= COALESCE(CAST(t1.enddate AS DATE), dr.dateOnly) AND
```

Comment:
This is based on taste and what I think is readability, but I would do an explicit test for NULL rather than COALESCE() it. The intention behind COALESCE() doesnt seem as clear to me, if someone reads this in the future. so something like:
```
ON (dr.dateOnly >= t1.dateOnly AND (dr.dateOnly <= t1.enddate OR t1.enddate IS NULL))
```

Reply:
@bbimber, thanks for reviewing the code. I prefer your suggestion as it's cleaner and self-documenting the issue.